### ISSUE TITLE
items 8310 are mainly false positives

### ISSUE THREAD

**Issue by zorglubu** [osm-fr/osmose-backend#1019]
Hello,

I have tested around 30 items in an area where I live. I have not found even one POI to be inserted : only false positives.
https://osmose.openstreetmap.fr/fr/map/#zoom=16&lat=48.78343&lon=2.33922&item=8310&level=1%2C2%2C3&tags=&fixable=

The POI's are supposed to be shops. But I presume that these addresses are mainly private addresses of the owners of a shop. Is it possible to remove juridic entities  from the SIRENE base ? If not, it seems that these opened data are not exploitable for OSM.

Amicalement.

**Comment by frodrigo** [comment]
If you find some pattern in the false positives we can improve the filters. Remove shop type or add minimum employe to avoid owner addresses.

**Comment by zorglubu** [comment]
This is why I tried to find some real notification (missing shop). In order to find a way to keep them in the filter. I finally found one real shop : a funeral_director. Difficult to find a pattern with that.
In another way I have tried to find a 2 bakeries and a butcher in https://entreprise.data.gouv.fr/, I don't find them with the name of the street.
I have the feeling that these data are not for OSM.  :-(
